proto/minder/v1/minder.proto

```diff
@@ -886,6 +886,35 @@ service ProvidersService {
     }
 }
 
+service InviteService {
+    rpc GetInviteDetails(GetInviteDetailsRequest) returns (GetInviteDetailsResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/invite/{code}"
+        };
+
+        option (rpc_options) = {
+            target_resource: TARGET_RESOURCE_PROJECT
+        };
+    }
+}
+
+message GetInviteDetailsRequest {
+    Context context = 1;
+    // Invite nonce/code to retrieve details for
+    string code = 2;
+}
+
+message GetInviteDetailsResponse {
+    // Project associated with the invite
+    string project_display = 1;
```

Comment:
is this only the project's display name? don't we want to show the actual project's name or ID?

Reply:
Display name is enough in this case since this API will be used solely to query invite metadata based on the invite code so the FE can fill things like "X invited you to join Y"

Reply:
And we'd be persisting the project elsewhere, right?

Reply:
Yes 👍